### DISCUSSION TITLE
Add Code - Upload Screenshot with QR code to import.

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#2810] [Enhancements]
Along with Manually Copy Pasting Codes and Scanning QR Codes, want option to Upload Screenshot or QR code image to recognize and impprt keys. Snip attached from 2FAS which has this feature. 
![Uploading Picsart_24-08-22_08-33-24-402.jpg…]()

**Comment by ua741** [comment]
@haritvsmurali  Please search existing issue before creating new issues
Duplicate of https://github.com/ente-io/ente/discussions/487
